src/net/wooga/jenkins/pipeline/model/Gradle.groovy

```diff
@@ -30,9 +30,9 @@ class Gradle {
     def wrapper(String command, Boolean returnStatus = false,
                                 Boolean returnStdout = false) {
         if (jenkins.isUnix()) {
-            return jenkins.sh(script: "./gradlew ${formatCommand(command)}", returnStdout: returnStdout, returnStatus: returnStatus)
+            return jenkins.sh(script: "./gradlew --no-daemon ${formatCommand(command)}", returnStdout: returnStdout, returnStatus: returnStatus)

```

Comment:
Why the `--no-daemon`? This may end up impacting build times, which are already somewhat high for WDKs.

Reply:
In general the deamon should be disabled already for CI builds. It was suggested in the old gradle docs to do that for CI though. But I would not put this as a general switch to all gradle calls like this.